CHANGELOG.md

```diff
@@ -38,7 +38,6 @@
 - Add `cardinality_limited` outcome with id `6`. ([#3389](https://github.com/getsentry/relay/pull/3389))
 - Extract `cache.item_size` and `cache.hit` metrics. ([#3371]https://github.com/getsentry/relay/pull/3371)
 - Optionally convert segment spans to transactions for compatibility. ([#3375](https://github.com/getsentry/relay/pull/3375))
-- Add feature flag for replay video event types. ([#3402](https://github.com/getsentry/relay/pull/3402))
```

Comment:
24.4.0 was already released, so we should not remove features from the changelog for it.

Reply:
Should I open a new PR with the changelog entry?

Reply:
Yes please.